Monika After Story/game/styles.rpy

```diff
@@ -388,3 +388,326 @@ init -10 python in mas_ui:
             if data["return_value"] == data["true_value"]:
                 return selected_prompt
         return default_prompt
+
+    # Methods for twopane_scrollable_menu
+    TWOPANE_MENU_MAX_FLT_ITEMS = 50
+    TWOPANE_MENU_SEARCH_DBS = (
+        store.mas_all_ev_db_map["EVE"].values()
+        + store.mas_all_ev_db_map["BYE"].values()
+        + store.mas_all_ev_db_map["STY"].values()
+        + store.mas_all_ev_db_map["CMP"].values()
+        + store.mas_all_ev_db_map["SNG"].values()
+    )
+    TWOPANE_MENU_DELEGATES_CALLBACK_MAP = {
+        "mas_compliment_": store.mas_compliments.compliment_delegate_callback
+    }
```

Comment:
I understand why this was done, but I'm not sure it's the best idea. I can see how we can run code that could be missed when jumping directly to a topic, but some cases (like the farewell's jump to `bye_long_absence_2`) would be missed. And if we did add the jump to a delegate function, it wouldn't make that much sense to the user if they tried to jump to a farewell only to get thrown into a different one.

Additionally, since the additional topics aren't normally accessible in the two pane talk menu, having them only be visible when searching is inconsistent.

A general search bar/jump to topic bar might be a better option for searching over all topics, but again, skipping over delegate code could still have problems, so can't say its something we should do right now.

Reply:
It feels nice to skip the delegate label when you want to tell her a compliment. The point of this is to save time looking for the topic you want to push. So while it can be inconsistent from the two pane screen, it's more convenient in its own format. 

Although, I think we don't need to have farewells there since they go under their own prompt in the talk menu.

Reply:
keeping it regular topics and compliments sounds good to me